### ISSUE TITLE
about the dataset augmentation of mixup

### ISSUE THREAD

**Issue by Ethan-niu** [ultralytics/yolov5#7038]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

First thanks for your great job!
When I saw the code of mixup , I found the image has used the lambda, but the label dosen't, I want to know why, thank you

### Additional

def mixup(im, labels, im2, labels2):
    # Applies MixUp augmentation https://arxiv.org/pdf/1710.09412.pdf
    r = np.random.beta(32.0, 32.0)  # mixup ratio, alpha=beta=32.0
    im = (im * r + im2 * (1 - r)).astype(np.uint8)
    labels = np.concatenate((labels, labels2), 0)
    return im, labels

**Comment by glenn-jocher** [comment]
@Ethan-niu labels are present in both images used in mixup. There is no such thing as a label weight.

**Comment by Ethan-niu** [comment]
> @Ethan-niu labels are present in both images used in mixup. There is no such thing as a label weight.

But in the classification when used mixup, the loss= r*loss1 + (1-r)*loss2, if the classification and detection have any difference

**Comment by glenn-jocher** [comment]
@Ethan-niu that's classification though, this is object detection and the anchor location will only have 1 object.
